about.html

```diff
@@ -0,0 +1,163 @@
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Burning Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">The Burning Circle Flight Academy</h1>
+      <div class="first-section-para">
+        <p>
+          The Burining Circle Flight Academy airshow is one of Zambia's first airshows 
+          to be hosted by a privately owned aviation institution. The whole essence of 
+          this event is to connect aerospace industry proffessionals across Africa and
+          the world at Large. This is also a great opportunity for youths interested in
+          aviation and becoming Pilots to learn from the best Pilots and know what it takes
+          to become a world class pilot. 
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
+        
+        </p>
+        <p class="email-address">
+          burningcircleflightacademy@gmail.com
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="flex-content">
+      <h2> The Burning Circle Flight Academy Airshow</h2>
+      <hr>
+      <p class="mobile-para">This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+      <p class='desktop-para'> This will be the very first Airshow event to be hosted by Burning circle Flight academy the event will run from November 14th - 18th, so your full particpation and support will be appreciated.</p>
+      <img src="images/plane3.svg" alt="plane" class="plane">
+    </div>
+    <hr class="horizontal-line">
+      
+  </section>
+
+  <section class="past-section">
+    <div class="past-section-container flex-past">
+      <h2>Past Airshow events</h2>
+      <hr>
+      <p>Take a look at the past airshows we attended and what inspired us to host our own Airshow event as a Pilot training institution.</p>
+      <div class="pics">
+      <img src="images/airshow1.jpeg" alt="Dubai airshow2019">
+      <img src="images/airshow2.jpeg" alt="Dubai airshow2020">
+    </div>
```

Comment:
- [ ] As suggested by the previous reviewer, you should add an orange mask over these images, as well as centered text, as you can see in the behance design: 
 
![image](https://user-images.githubusercontent.com/66967274/142002202-a5db3982-17d5-477d-a285-fbe9f3e32f4c.png)

Your design: 
![image](https://user-images.githubusercontent.com/66967274/142002615-6b9d7b6a-feab-46e6-b967-df026d5c6adc.png)

Reply:
Hello Vanessa, how do i go about doing that??

Reply:
Hi, you can read more about the mask in [this tutorial](https://dev.to/ellen_dev/two-ways-to-achieve-an-image-colour-overlay-with-css-eio), for the text, you can read more about in [this other article](https://www.w3schools.com/howto/howto_css_image_text_blocks.asp).

Hope it helps! Happy coding 👨‍💻